minimal-notebook/start-notebook.sh

```diff
@@ -12,5 +12,4 @@ if [ ! -z "$GRANT_SUDO" ]; then
 fi
 
 # Start the notebook server
-exec su $NB_USER -c "env PATH=$PATH jupyter notebook $@"
-
+exec su $NB_USER -c "env PATH=$PATH jupyter notebook $*"
```

Comment:
Yes, though I believe the first form tries to quote each individual item, but perhaps not properly within the context of the quotes already in the start-notebook script. I was seeing any arguments beyond $1 winding up as parameters to `su`, not to `jupyter notebook`.